### PR TITLE
ActionSheetStringMultipleSelectionPicker

### PR DESCRIPTION
**Pull request by oblador**
This PR builds on top of (the dead) #264 and:
* Fixes the requested changes in that PR
* Fixes a bug where the `selectedValue` argument weren't correct
* Adds new files in `Components` to podspec

### CODE REVIEW

**Comment by chirag04** [comment]
@skywinder

**Comment by skywinder** [comment]
@oblador wow, looks great! Really sorry for late reply. 
Can you also add the example with new picker to sample project?

**Comment by oblador** [comment]
Sorry, but I don't have the time to do it right now. Merging without is up to you :-)

**Comment by chirag04** [comment]
@skywinder i can follow up with a PR that adds an example.

**Comment by ibrahimyilmaz7** [comment]
Can't found this "ActionSheetStringMultipleSelectionPicker". Is this still supported?

**Comment by skywinder** [comment]
I'd be more than happy to merge a PR adding this feature, it seems like it would be useful for sure.
It's would be very good to test it at least in sample project before adding to the `master`.

Since there ling to no activity: Feel free to reopen the issue if you would like to finish it. 👍

**Comment by oblador** [comment]
@skywinder I've fixed conflicts. Any chance this would be merged or that you would help in getting the last bits in?

**Comment by skywinder** [comment]
Hello, @oblador . Thank you very much and sorry for the late reply. 
I'm back in the project and want to restart it again. 
Looks good. Let's add the example of new picker in same way as all the others has. And I will merge this PR.

File: ActionSheetPicker-3.0.podspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do | s |
     s.requires_arc = true
     s.ios.deployment_target = '6.1'
     s.platform = :ios
-    s.public_header_files = 'ActionSheetPicker.h', 'Pickers/*.h'
-    s.source_files = 'ActionSheetPicker.h', 'Pickers/*.{h,m}'
+    s.public_header_files = 'ActionSheetPicker.h', 'Pickers/{,Components/}*.h'
+    s.source_files = 'ActionSheetPicker.h', 'Pickers/{,Components/}*.{h,m}'
```
Comment (skywinder) [inline ActionSheetPicker-3.0.podspec]: Does it changes something?
I believe you have to add your picker to `ActionSheetPicker-3.0/CoreActionSheetPicker/CoreActionSheetPicker/CoreActionSheetPicker.h` instead

have a look: https://github.com/skywinder/ActionSheetPicker-3.0/blob/c78d7a8af26eb9e5f22ecb849c54804d6fd31be1/CoreActionSheetPicker/CoreActionSheetPicker/CoreActionSheetPicker.h

**Comment by skywinder** [comment]
@oblador I collect all kind of pickers in the exmpale project. (https://github.com/skywinder/ActionSheetPicker-3.0#example-projects)
Can you please add example to the example project of your picker?

**Comment by oblador** [comment]
I’m not using this library anymore, the PR was opened three years ago

**Pull request closed without merging**